src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -1,11 +1,6 @@
 package jenkins.plugins.slack;
 
-import hudson.Util;
-import hudson.model.Result;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
-import hudson.model.CauseAction;
-import hudson.model.Run;
+import hudson.model.*;
```

Comment:
Wildcard imports increase the difficulty of troubleshooting and should be avoided.  It's better to call out each class individually as you import them.  This makes tracking down issues and knowing what classes come from which package simple.